plugin/src/main/java/com/denizenscript/denizen/objects/MaterialTag.java

```diff
@@ -671,6 +671,19 @@ public static void register() {
                 return itemType.isEdible() ? new ElementTag(NMSHandler.itemHelper.getFoodPoints(itemType)) : null;
             });
         }
+
+        if (NMSHandler.getVersion().isAtLeast(NMSVersion.v1_19)) {
+
+            // <--[tag]
+            // @attribute <MaterialTag.is_enabled[<world>]>
+            // @returns ElementTag(Boolean)
+            // @description
+            // Returns whether the material is enabled in the specified world.

```

Comment:
Docs should clarify better what it means to be enabled: when would something be enabled vs disabled, and what does disabling actually _do_?